CHANGES.md

```diff
@@ -1,5 +1,50 @@
 # Changes
 
+## edge-23.1.1
+
+This edge release introduces a number of different fixes changes to the proxy.
+The proxy has been updated to initialize routes lazily, which means service
+profile routes will now only show up in the metrics when a route is used. In
+the extensions, old (`ServerAuthorization`) resources have been converted to
+`AuthorizationPolicy` -- as part of this change, redundant policy resources
+have been cleaned up. A bug in the destination controller that could
+potentially lead to stale pods being considered in the load balancer has been
+fixed; operations that could previously result in this behavior are now
```

Comment:
it may be too "in the weeds" to talk about these operations being infallible now.  I think it's sufficient to say that the bug was fixed.

Reply:
👍🏻 makes sense.